sanic_routing/tree.py

```diff
@@ -191,15 +210,24 @@ def _inject_params(self, location, indent):
             lines
             + [
                 Line("except ValueError:", indent),
-                Line("...", indent + 1),
+                Line("pass", indent + 1),
```

Comment:
why did this need to be changed from `...` to `pass`? Is it to avoid conflict with the reindentation lines?

Reply:
Yup. I don't think that it was an issue. Sort of a proactive change.